### PR TITLE
Align delivery-tracking/shipment-info with conventions

### PR DESCRIPTION
**Pull request by janhalama**
## Description

This PR aligns delivery-tracking/shipment-info use-case with [Station conventions](https://github.com/superfaceai/station/blob/main/CONVENTIONS.md) and turn tests to use Superface Testing.

## Motivation and Context

Shippo and DHL tokens are public test tokens and we can have them in recordings.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I haven't repeated the code. (DRY)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: capabilities/delivery-tracking/shipment-info/profile.supr [hunk 1]
```diff
@@ -1,90 +1,178 @@
...
-    trackingNumber! string
-
-    "Carrier
-    Shipment carrier identification"
-    carrier string
+    """
+    Shipment tracking number
+    The identifier of shipment.
+    """
+    trackingNumber! string!
```
[the remark this answers is not in the document]
Comment (janhalama) [reply capabilities/delivery-tracking/shipment-info/profile.supr:19]: `null` value passed in tracking number will cause one-sdk to fail fast and return validation error. Previous version would send the request. Both providers returns HTTP 400 status code in that case.

The error that is returned changed. Is not it even major change.

File: capabilities/delivery-tracking/shipment-info/profile.supr [hunk 2]
```diff
@@ -1,90 +1,178 @@
...
+"""
 model Location {
-  address! Address
-}
-
-model Address {
-
-  "Country code
-   A short text string code (ISO 3166-1 alpha-2 country code) specifying the country."
+  address! {
```
[the remark this answers is not in the document]
Comment (janhalama) [reply capabilities/delivery-tracking/shipment-info/profile.supr:158]: Thank you for the catch. Fixed.

File: capabilities/delivery-tracking/shipment-info/profile.supr [hunk 3]
```diff
@@ -1,90 +1,178 @@
...
-"
+The status of the shipment. Harmonized across different carriers.
+"""
 model ShipmentStatus enum {
+  """
+  Pre-Transit
+  Carrier has received information about the package but it has not yet been scanned and picked up.
+  """
   pre_transit
+  """
```
[the remark this answers is not in the document]
Comment (janhalama) [reply capabilities/delivery-tracking/shipment-info/profile.supr]: Fixed.

**Review by freaz — APPROVED** [review]
Looks good now, thanks. 🚢

**Pull request merged**